test/e2e_node/security_context_test.go

```diff
@@ -364,77 +364,50 @@ var _ = framework.KubeDescribe("Security Context", func() {
 							Name:    podName,
 							Command: command,
 							SecurityContext: &v1.SecurityContext{
-								RunAsUser: &userid,
+								Privileged: &privileged,
 							},
 						},
 					},
 				},
 			}
 		}
-		createAndWaitUserPod := func(userid int64) {
-			podName := fmt.Sprintf("busybox-user-%d-%s", userid, uuid.NewUUID())
+		createAndWaitUserPod := func(privileged bool) string {
+			podName := fmt.Sprintf("busybox-privileged-%v-%s", privileged, uuid.NewUUID())
 			podClient.Create(makeUserPod(podName,
 				busyboxImage,
-				[]string{"sh", "-c", fmt.Sprintf("test $(id -u) -eq %d", userid)},
-				userid,
+				[]string{"sh", "-c", "ip link add dummy0 type dummy || true"},
+				privileged,
 			))
 
 			podClient.WaitForSuccess(podName, framework.PodStartTimeout)
+
+			return podName
 		}
 
-		It("should run the container with uid 65534 [NodeConformance]", func() {
-			createAndWaitUserPod(65534)
-		})
+		It("should run the container as privileged when true [NodeFeature:HostAccess]", func() {
+			podName := createAndWaitUserPod(true)
+			logs, err := framework.GetPodLogs(f.ClientSet, f.Namespace.Name, podName, podName)
+			if err != nil {
+				framework.Failf("GetPodLogs for pod %q failed: %v", podName, err)
+			}
 
-		It("should run the container with uid 0 [NodeConformance]", func() {
-			createAndWaitUserPod(0)
+			framework.Logf("Got logs for pod %q: %q", podName, logs)
+			if strings.Contains(logs, "Operation not permitted") {
+				framework.Failf("privileged container should be able to create dummy device")
+			}
 		})
-	})
 
-	Context("When creating a pod with readOnlyRootFilesystem", func() {
-		makeUserPod := func(podName, image string, command []string, readOnlyRootFilesystem bool) *v1.Pod {
-			return &v1.Pod{
-				ObjectMeta: metav1.ObjectMeta{
-					Name: podName,
-				},
-				Spec: v1.PodSpec{
-					RestartPolicy: v1.RestartPolicyNever,
-					Containers: []v1.Container{
-						{
-							Image:   image,
-							Name:    podName,
-							Command: command,
-							SecurityContext: &v1.SecurityContext{
-								ReadOnlyRootFilesystem: &readOnlyRootFilesystem,
-							},
-						},
-					},
-				},
+		It("should run the container as unprivileged when false [NodeConformance]", func() {
```

Comment:
Is this PR moving all [NodeConformance] tests to e2e/common? Should this and other remaining [NodeConformance] tests be moved?

Reply:
Given the [comments from Filipe](https://github.com/kubernetes/kubernetes/pull/67032#issuecomment-411187287), it seems to me that we can remove the container runtime constraint.

And why is privileged mode a problem?

Reply:
>Given the comments from Filipe, it seems to me that we can remove the container runtime constraint.

Is there any chances that few k8s workloads might be running on CentOS < 7 or running with the lower container-selinux version where the [mentioned patch](https://github.com/containers/container-selinux/issues/45) is not added?

- If this is not a concern, I can create new PR to add [remaining [NodeConformance] tests](https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/security_context_test.go#L500-L521) validating logs under AllowPrivilegeEscalation PodSpec. Removing [ContainerRuntime constraint](https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/security_context_test.go#L443-L464).

>And why is privileged mode a problem?

Privileged mode is not a problem. It's my bad, I missed to notice.
- New PR would include [remaining e2e tests](https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/security_context_test.go#L558-L582), validating logs under privileged PodSpec. Removing [[NodeFeature:HostAccess]](https://github.com/kubernetes/kubernetes/blob/55859a60fe09c412e183c92ad265cf0d52fbe3d8/test/e2e_node/security_context_test.go#L558:56-65) from test name

Reply:
> Is there any chances that few k8s workloads might be running on CentOS < 7 or running with the lower container-selinux version where the mentioned patch is not added?
> - If this is not a concern, I can create new PR to add remaining [NodeConformance] tests validating logs under AllowPrivilegeEscalation PodSpec. Removing ContainerRuntime constraint.

+@filbranden

> Privileged mode is not a problem. It's my bad, I missed to notice.
> - New PR would include remaining e2e tests, validating logs under privileged PodSpec. Removing [NodeFeature:HostAccess] from test name

Do you mean you will move those in a separate PR? Only the tests marked with [NodeConformance] should be moved. Let's discuss in the followup PR on whether we should change [NodeFeature:HostAccess] to [NodeConformance].

Reply:
> Is there any chances that few k8s workloads might be running on CentOS < 7 or running with the lower container-selinux version where the mentioned patch is not added?
> * If this is not a concern, I can create new PR to add remaining [NodeConformance] tests validating logs under AllowPrivilegeEscalation PodSpec. Removing ContainerRuntime constraint.

Fine to proceed with that.

If tests are run on CentOS, that should be on CentOS 7 with all the latest packages and that includes an appropriate container-selinux version.